rules/windows/process_creation/process_creation_msdeploy.yml

```diff
@@ -0,0 +1,34 @@
+title: Msdeploy.exe LOLBIN
+status: experimental
+id: d80d5c81-04ba-45b4-84e4-92eba40e0ad3
+author: Beyu Denis, oscd.community
+date: 2020/10/18
+description: launch binary via msdeploy.exe
```

Comment:
Consider a more elaborated description, like "Detects file execution using the msdeploy.exe lolbin"